lighthouse-core/test/audits/installable-manifest-test.js

```diff
@@ -50,26 +52,30 @@ describe('PWA: webapp install banner audit', () => {
 
       return InstallableManifestAudit.audit(artifacts, context).then(result => {
         assert.strictEqual(artifacts.WebAppManifest.url, EXAMPLE_MANIFEST_URL);
-        assert.strictEqual(result.details.items[0].manifestUrl, EXAMPLE_MANIFEST_URL);
+        const debugData = result.details.debugData;
+        assert.strictEqual(debugData.manifestUrl, EXAMPLE_MANIFEST_URL);
       });
     });
 
     it('fails with a non-parsable manifest', () => {
       const artifacts = generateMockArtifacts('{,:}');
+      artifacts.InstallabilityErrors.errors.push({errorId: 'manifest-empty', errorArguments: []});
```

Comment:
this is the same test as the one below (both `manifest-empty`). Since this is a test of a "non-parsable manifest", drop this line and it should trigger the `manifestValues.isParseFailure` check?